### ISSUE TITLE
Retain ref to template project so clones can pull in changes later.

### ISSUE THREAD

**Issue by reergymerej** [reergymerej/start#12]
Is this a good idea?

This tool is a springboard.  Once it's started, I don't care what template it came from.  It's likely that changes to the template won't really be something that would apply to an app that already started.

**Comment by kondor6c** [comment]
Yeah I agree, plus you might be concerned about overwriting content and then you'd have to introduce a versioning scheme to reflect breaking changes and what not. What about having this tool track which templates were created with itself. Changes could be passed down as a specific action, like with a `--migrate`. Like a manifest and perhaps a simple md5 of the files so that it can know if it needs to apply those changes. Or maybe I'm over engineering an simple utility?

**Comment by kondor6c** [comment]
A couple good examples I thought, both are different scenario's:

1. License

  - you really don't want a migration to effect a change from an "unlicense" to a ISC or BSD license. Doing so could involve legal difficulties and just a headache all around.

2. Package upgrade
 - if database driver had a critical zero day that needed patching and was in the Dockerfile that was used for the container image, updating that could be a lengthy task, updating all of them could be done, in an automated fashion like `find ~ -iname 'Dockerfile' |xargs -P 8 -I {} sed -ie 's/pgsql-v44.2/pgsql-v99.8/g' ` but it could pose risks and is able to be missed.
